x-pack/platform/plugins/shared/task_manager/server/config.ts

```diff
@@ -106,6 +113,10 @@ export const configSchema = schema.object(
     /* Allows for old kibana config to start kibana without crashing since ephemeral tasks are deprecated*/
     ephemeral_tasks: schema.maybe(schema.any()),
     event_loop_delay: eventLoopDelaySchema,
+    invalidate_api_key_task: schema.object({
+      interval: schema.string({ validate: validateDuration, defaultValue: '5m' }),
+      removalDelay: schema.string({ validate: validateDuration, defaultValue: '1h' }),
```

Comment:
Same as alerting, we introduce a removal delay to remove the possibility of race conditions between the task that's being deleted and any other child tasks that may be scheduled using the same API key.